etc/gn/openthread.gni

```diff
@@ -55,6 +55,9 @@ declare_args() {
   # Use external mbedtls. If blank, internal mbedtls will be used.
   openthread_external_mbedtls = ""
 
+  # Enable building for POSIX platforms.
+  openthread_posix = (current_os == "linux" || current_os == "fuchsia")
```

Comment:
Is fuchsia POSIX?

Reply:
It isn't although it is building as if it were Android for OpenThread. I think you are right that it's safer to leave this off there, though, removed it.